.gitlab-ci.yml

```
@@ -13,6 +13,7 @@ include:
 variables:
   DOTNET_PACKAGE_VERSION:
     description: "Used by the package stage when triggered manually"
+  REPO_NOTIFICATION_CHANNEL: "#apm-dotnet-error-bots"
```

I think `#apm-dotnet-bots` _may_ be the better channel (`#apm-dotnet-error-bots` is for prod errors)
```suggestion
  REPO_NOTIFICATION_CHANNEL: "#apm-dotnet-bots"
```

---

😆 

I was literally going to comment that I thought `#apm-dotnet-error-bots` may be a better one even though that yes it is supposed to be for prod errors.

My only concern is that `#apm-dotnet-bots` can be a tad noisy so it may slip through unnoticed there, but maybe not

I say we can try it and if they fail and we don't notice them then we can re-route.